roles/matrix-synapse/defaults/main.yml

```diff
@@ -641,3 +644,124 @@ matrix_appservice_irc_configuration_extension_yaml: |
 matrix_appservice_irc_configuration_extension: "{{ matrix_appservice_irc_configuration_extension_yaml|from_yaml if matrix_appservice_irc_configuration_extension_yaml|from_yaml else {} }}"
 
 matrix_appservice_irc_configuration: "{{ matrix_appservice_irc_configuration_yaml|from_yaml|combine(matrix_appservice_irc_configuration_extension, recursive=True) }}"
+
+
+# Matrix Appservice Discord is a Matrix <-> Discord bridge
+# Enable Discord bridge
+matrix_appservice_discord_enabled: false
+
+matrix_appservice_discord_docker_image: "halfshot/matrix-appservice-discord:latest"
+
+matrix_appservice_discord_base_path: "{{ matrix_base_data_path }}/appservice-discord"
+
+matrix_appservice_discord_client_ID: "YOUR DISCORD APP CLIENT ID"
```

Comment:
Since all other variables used by the playbook are in all-lowercase, it's probably better if we change this to lowercase as well (`matrix_appservice_discord_client_id`).

Reply:
changed. https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/105/commits/2d78c5f89d1e20f11e42ed4e6df44a1323b26ceb